cpp/ql/lib/semmle/code/cpp/controlflow/LocalScopeVariableReachability.qll

```diff
@@ -4,125 +4,6 @@
 
 import cpp
```

Comment:
`bbSuccessorEntryReachesLoopInvariant` is definitely just an implementation detail that should have never been public. I think this predicate is public because:
- We have a file `StackVariableReachability.qll` that is pretty much identical to file this, except that its API uses `StackVariable` instead of `LocalScopeVariable`.
- `StackVariableReachability.qll`'s `bbSuccessorEntryReachesLoopInvariant` predicate is used internally by [DefinitionsAndUses.qll](https://github.com/github/codeql/blob/main/cpp%2Fql%2Flib%2Fsemmle%2Fcode%2Fcpp%2Fcontrolflow%2FDefinitionsAndUses.qll#L211)

All in all, no one in their right mind would use `bbSuccessorEntryReachesLoopInvariant` from this file. I say we just delete it. I think the entire file can be replaced with:
```ql
import semmle.code.cpp.controlflow.StackVariableReachability
class LocalScopeVariableReachability = StackVariableReachability
```
As a bonus, this will also ensure that the `bbSuccessorEntryReachesLoopInvariant` predicate from `StackVariableReachability.qll` is still available when you import `LocalScopeVariableReachability.qll`.

Reply:
The `LocalScopeVariableReachability` class a deleted deprecation anyway, so we don't need that alias.  
I think the conclusion is that I can just delete the file?